server/members/method.js

```diff
@@ -0,0 +1,17 @@
+Meteor.methods({
+  editStatus: function(data) {
+    check(data, {
+      user_id: String,
+      status: String,
+      study_group_id: String
+    });
+
+    if (!this.userId) {
+      throw new Meteor.Error('Members.methods.editStatus.not-logged-in', 'Must be logged in to edit the member status.');
+    }
+    StudyGroups.update({_id: data.study_group_id, 'members.id': data.user_id },
```

Comment:
while modifying info related to current user, we should either use `Meteor.user()` or `Meteor.userId()` for security reasons. `'members.id': Meteor.userId()  },`

Reply:
fixed